google/cloud/spanner_v1/streamed.py

```diff
@@ -17,9 +17,11 @@
 from google.cloud import exceptions
 from google.cloud.spanner_v1 import TypeCode
 import six
+import logging
```

Comment:
Nit: It doesn't appear as though `logging` is used in this module. Even if it was, the `logging` module is a built-in and belongs in an import group separate from the third-party modules.